bin/install.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+set -eu
+
+# Ensure PHP is installed
+if ! command -v php > /dev/null 2>&1; then
+  echo "PHP is not found, installation aborted!"
+  exit 1
+fi
+
+# Ensure Composer is installed/updated
+if ! command -v composer > /dev/null 2>&1; then
+  echo "*** Installing Composer ***"
+  php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"
+
+  if [ \
+      "$(curl -S https://composer.github.io/installer.sig)" = \
+      "$(php -r \"echo hash_file\('SHA384', 'composer-setup.php'\);\")" \
+  ]; then
+      php composer-setup.php --quiet --filename=composer
+      sudo mv composer /usr/local/bin/composer
```

Comment:
It has been decided: 
<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr"><a href="https://twitter.com/8BallTweets">@8BallTweets</a> Should we use `sudo mv`?</p>&mdash; 422 - Unprocessable (@potherca) <a href="https://twitter.com/potherca/status/791019040613470208">October 25, 2016</a></blockquote>

<blockquote class="twitter-tweet" data-lang="en"><p lang="und" dir="ltr"><a href="https://twitter.com/potherca">@potherca</a> - Yes</p>&mdash; Ask the Magic 8 Ball (@8BallTweets) <a href="https://twitter.com/8BallTweets/status/791019196452921346">October 25, 2016</a></blockquote>